src/Layouts/Layout.php

```diff
@@ -309,4 +310,14 @@ protected function getProcessedKey($key)
 
         return substr(bin2hex($bytes), 0, 16);
     }
+    
+    /**
+     * Convert the model instance to an array.
+     *
+     * @return array
+     */
+    public function toArray()
+    {
+        return array_merge($this->attributesToArray(), $this->relationsToArray());
```

Comment:
What is the reason for adding `$this->relationsToArray()` in here? There are no relations on flexible layouts, and it threw an exception during our testing.

After replacing that line with `return $this->attributesToArray();`, it works properly.

Do you have a use-case where you need to include the relations on a flexible layout, somehow?

Reply:
The `$this->attributesToArray()` method throws an exception because it is calling `getVisible` and  `getHidden` behind the scenes which are included in the `HidesAttributes` trait from the Illuminate source. This class currently does not `use HidesAttributes`, but adding it fixes the problem.

Reply:
@voidgraphics first of all thanks for your feedback.

to be honest i cannot remember right now. i have to double check my WIP project.

You can decide which route we go, there is a high percentage the `relationsToArray()` is unnecessary.